src/fender_value.rs

```diff
@@ -21,6 +21,7 @@ pub enum FenderValue {
     List(InternalReference<Vec<FenderReference>>),
     Struct(FenderStruct),
     Type(FenderTypeId),
+    HashMap(InternalReference<HashMap<FenderValue, InternalReference<FenderReference>>>),
```

Comment:
Why does the key type need to be InternalReference<FenderReference> instead of just FenderReference?

Reply:
key type is `FenderValue`

Reply:
Meant value type, sorry